### PR TITLE
Improve code coverage in Geochemistry

### PR DESCRIPTION
**Pull request by WilkAndy**
This commit concerns coverage of:

GeochemicalDatabaseReady
EquilibriumConstantInterpolator
GeochemistryActivityCoefficientsDebyeHuckel

Refs #17134

### CODE REVIEW

File: modules/geochemistry/include/utils/GeochemicalDatabaseReader.h [hunk 1]
```diff
@@ -522,6 +512,19 @@ class GeochemicalDatabaseReader
   GeochemistryDebyeHuckel _debye_huckel;
   /// Neutral species activity coefficients
   std::map<std::string, GeochemistryNeutralSpeciesActivity> _neutral_species_activity;
-  // Helper for converting json node to Real from string, if needed
+  // Helper for converting json node to Real from string, if needed (see
+  // https://www.urbandictionary.com/define.php?term=get%20real)
```
Comment (loganharbour) [inline modules/geochemistry/include/utils/GeochemicalDatabaseReader.h:516]: !?

Comment (WilkAndy) [reply modules/geochemistry/include/utils/GeochemicalDatabaseReader.h:516]: That is the product of too much beer.  How did you see that line amongst everything else?!

**Review by loganharbour — APPROVED** [review]

File: modules/geochemistry/include/utils/GeochemicalDatabaseReader.h (hunk 1 above)
Comment (loganharbour) [reply modules/geochemistry/include/utils/GeochemicalDatabaseReader.h]: Review does actually mean review 😆

**Comment by WilkAndy** [comment]
Thanks @loganharbour :-)

**Pull request merged**